bcda/api/alr_test.go

```diff
@@ -32,24 +30,9 @@ func TestAlrTestSuite(t *testing.T) {
 }
 
 func (s *AlrTestSuite) SetupSuite() {
-	// See testdata/acos.yml
-	s.acoID = uuid.Parse("a7ff9610-0977-4a90-867e-f6b2b4c8b6a8")
-	s.db, _ = databasetest.CreateDatabase(s.T(), "../../db/migrations/bcda/", true)
-	tf, err := testfixtures.New(
-		testfixtures.Database(s.db),
-		testfixtures.Dialect("postgres"),
-		testfixtures.Directory("testdata/"),
-	)
-
-	if err != nil {
-		assert.FailNowf(s.T(), "Failed to setup test fixtures", err.Error())
-	}
-	if err := tf.Load(); err != nil {
-		assert.FailNowf(s.T(), "Failed to load test fixtures", err.Error())
-	}
-
-	enqueuer := &queueing.MockEnqueuer{}
-	enqueuer.On("AddAlrJob", mock.Anything, mock.Anything).Return(nil)
+	// UUID for ACO Dev
+	s.acoID = uuid.Parse("0c527d2e-2e8a-4808-b11d-0fa06baf8254")
+	s.db = database.Connection
```

Comment:
While testfixture is a great tool, removed it and used actual synthetic ALR data to test.